doc/source/release-notes/Percona-Server-8.0.28-20.rst

```diff
@@ -0,0 +1,35 @@
+.. _8.0.28-19:
+
+================================================================================
+*Percona Server for MySQL* 8.0.28-20 (2022-06-)
+================================================================================
+
+.. include:: ../_res/rn/ps-mysql-blurb.txt
+
+.. contents::
+   :local:
+
+.. include:: ../_res/rn/8.0.28-highlights.txt
+
+Improvement
+=================================================
+
+* :jirabug:`PS-8103`: Memory leak and errors detectors (Valgrind or AddressSanitizer) provide detailed stack traces from dynamic libraries (plugins and components). The detailed stack traces make it easier to identify and fix the issues.
+
```

Comment:
We're missing the announcement of libzbd 2.1.0 and ZenFS fixes (WDCZEN-27, WDCZEN-31, WDCZEN-29, and WDCZEN-32)

Reply:
Updated